apps/frontend/src/App.css

```
@@ -48,6 +48,16 @@ body {
   background: #6b7280;
 }
 
+/* Mobile drawer slide-in animation */
+@keyframes slideIn {
```

Stylelint (17.6.0)

[error] Expected keyframe name "slideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)